vehicle_name: auh
controllers:
  pid_heading:
    controller_type: pid
    controlled_variable: heading
    Kp: 0.5
    Ki: 0.01
    Kd: 1.6
    integral_limit: 1.0
    output_limit: 200.0
    output_reverse: true
    is_angle: true
    debug: true
  pid_depth:
    controller_type: pid
    controlled_variable: depth
    Kp: 32
    Ki: 0.01
    Kd: 48
    integral_limit: 6.0
    output_limit: 200.0
    output_reverse: false
    is_angle: false
    debug: true
  pid_speed:
    controller_type: pid
    controlled_variable: speed
    Kp: 39
    Ki: 0.01
    Kd: 74
    integral_limit: 8.0
    output_limit: 200.0
    output_reverse: false
    is_angle: false
    debug: true
